diff --git a/node_modules/@webtides/element-js/src/BaseElement.js b/node_modules/@webtides/element-js/src/BaseElement.js
index 189a14e..f633d01 100644
--- a/node_modules/@webtides/element-js/src/BaseElement.js
+++ b/node_modules/@webtides/element-js/src/BaseElement.js
@@ -69,13 +69,6 @@ class BaseElement extends HTMLElement {
 			propertyOptions: {},
 			...options,
 		};
-
-		if (this.hasAttribute('defer-update')) {
-			this._options.deferUpdate = true;
-		}
-		if (this.hasAttribute('defer-connected')) {
-			this._options.deferConnected = true;
-		}
 	}
 
 	/**
@@ -84,6 +77,13 @@ class BaseElement extends HTMLElement {
 	 * register observers, refs and events.
 	 */
 	connectedCallback() {
+		if (this.hasAttribute('defer-update')) {
+			this._options.deferUpdate = true;
+		}
+		if (this.hasAttribute('defer-connected')) {
+			this._options.deferConnected = true;
+		}
+
 		if (this._options.deferConnected) {
 			this._options.deferConnected = false;
 			return;
diff --git a/node_modules/@webtides/element-js/src/TemplateElement.js b/node_modules/@webtides/element-js/src/TemplateElement.js
index 4805be9..8ee6be8 100644
--- a/node_modules/@webtides/element-js/src/TemplateElement.js
+++ b/node_modules/@webtides/element-js/src/TemplateElement.js
@@ -24,8 +24,11 @@ class TemplateElement extends StyledElement {
 			...options,
 		});
 		this._template = this._options.template;
+	}
 
+	connectedCallback() {
 		if (this._options.shadowRender) this.attachShadow({ mode: 'open' });
+		super.connectedCallback();
 	}
 
 	/**
